src/coreclr/hosts/unixcorerun/corerun.cpp

```diff
@@ -484,3 +484,8 @@ int main(const int argc, const char* argv[])
                             managedAssemblyArgv);
     return exitCode;
 }
+
+int main(const int argc, const char* argv[])
+{
```

Comment:
Do we want to exclude the main method when this file is included by the sos project?